### PR TITLE
GoWatchIt: Refactor

### PR DESCRIPTION
**Pull request by MrChrisW**
Small changes to GoWatchIt #1463

---

_What's changed?_
- Use Text::Trim, lowercase using remainder_lc _(GoWatchIt.pm)_
- Remove CSS comments, Fix indentation _(go_watch_it.css)_
- Add additional comments to js _(go_watch_it.js)_

//CC @jagtalon

### CODE REVIEW

**Comment by moollaza** [comment]
LGTM :+1: 

Would like @jagtalon to sign off.

**Comment by jagtalon** [comment]
Awesome! Checking it out--thanks for this. And apologies that it doesn't work in DuckPAN at the moment because it's in the `assets/` dir

**Comment by MrChrisW** [comment]
This has been rebased due to #1470, ready for merge.
@jagtalon No problem, I managed to test it anyway :wink:

**Comment by jagtalon** [comment]
Hey, @MrChrisW--I'll just get this in (https://github.com/duckduckgo/zeroclickinfo-spice/pull/1477) and then merge this right away since it looks good! :)

**Comment by MrChrisW** [comment]
@moollaza Now that #1477 has been merged, this should be good to go.

**Comment by jagtalon** [comment]
Hey, @MrChrisW! Looks good except some of the CSS declarations have missing semicolons. I kinda like having the space between them too like `font-size: 1em` instead of `font-size:1em`, but I can't really say that because we don't have a set code styleguide. @moollaza maybe we should have one?

@MrChrisW so yeah just semicolons and it should be good to go! :D

File: share/spice/go_watch_it/go_watch_it.css [hunk 1]
```diff
@@ -13,51 +13,41 @@
...
-  height: 60px;
-  margin-top: 0.1em;
+    height: 60px;
+    margin-top: 0.1em
 }
 
 .tile--go_watch_it .tile__body .data span.price {
-  font-weight: bold;
-  font-size: 1.5em;
-  font-weight: 400;
```
Comment (MrChrisW) [inline share/spice/go_watch_it/go_watch_it.css]: @jagtalon Any reason you have the two `font-weight` props here?

**Comment by MrChrisW** [comment]
@jagtalon I agree with you recommendations as spacing improves readability :+1: Updated!

File: share/spice/go_watch_it/go_watch_it.css (hunk 1 above)
Comment (jagtalon) [inline share/spice/go_watch_it/go_watch_it.css]: @MrChrisW oof that was a mistake. :)

File: share/spice/go_watch_it/go_watch_it.css [hunk 2]
```diff
@@ -80,14 +70,15 @@
...
-.tile--go_watch_it .tile__body .gwi-icon--alt {
+.dark-header .tile--go_watch_it .tile__body .gwi-icon {
     display: none;
 }
-    .dark-header .tile--go_watch_it .tile__body .gwi-icon--alt {
-        display: inline-block;
-    }
 
+.tile--go_watch_it .tile__body .gwi-icon--alt {
+    display: nonel
```
Comment (jagtalon) [inline share/spice/go_watch_it/go_watch_it.css]: Whoops, one small typo here.

**Pull request merged**